test_requirements.txt

```diff
@@ -1 +1,2 @@
-pyftpdlib==1.5.2
\ No newline at end of file
+pyftpdlib==1.5.2
+responses==0.7.0
```

Comment:
pyftpdlib was already in test_requirements, looks like it's only being used in tests. Responses is used to mock responses in tests only.